.appveyor.yml

```diff
@@ -8,7 +8,6 @@ environment:
   GOPATH: C:\gopath
   PATH: C:\go\bin;C:\tools\mingw64\bin;$(PATH)
   CGO_ENABLED: 1
-  GO111MODULE: off
```

Comment:
makefiles & any other calls to go build are going to need something like this: https://github.com/opencontainers/runc/blob/master/Makefile#L17-L20

Reply:
by default, the go 1.13 has GO111MODULE on, so we dont need to have it?

Reply:
in the run I looked at for git hub actions it was not on.. The build of containerd failed.. because it was trying to build containerd using go build.. which pulled down the wrong version of containerd.

Reply:
👍